### PR TITLE
Added pinned messages to end-user docs

### PR DESCRIPTION
**Pull request by JeffSchering**
This is for [PLT-4176: Implementation of Pinned Posts #4217](https://github.com/mattermost/platform/pull/4217)

### CODE REVIEW

**Comment by JeffSchering** [comment]
I think it's a bit of a grey area whether or not this fits into the messaging basics topic, but since I'm rewriting the whole User's Guide to be more task-based and goal-oriented anyway, I'll leave it out.

**Comment by jasonblais** [comment]
Sounds good, looks good otherwise 👍

File: source/help/messaging/pinning-messages.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+All members of a channel can pin important or useful messages to that channel. The list of pinned messages is visible to all channel members.
+
+
+Pinned messages are marked with the pinned icon, |pinned-icon|. For example:
+
+.. image:: ../../images/pinned-example-channel.png
+
+**To view the list of pinned messages:**
+
+To view the complete list of pinned messages, click the pin icon |pin-icon|, located at the top of the channel. The right hand sidebar opens to show the list of pinned messages. For example:
```
Comment (lindy65) [inline source/help/messaging/pinning-messages.rst]: `... right hand sidebar...` > `right-hand-side bar` ?

Comment (JeffSchering) [inline source/help/messaging/pinning-messages.rst]: Thanks! I'll change it to Right-Hand Sidebar as that's what's used elsewhere.

**Pull request merged**